liota/dccs/iotcc.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, con):
         time.sleep(0.5)
         self.proto = HelixProtocol(self.comms, self.comms.identity.username, self.comms.identity.password)
         self._iotcc_json = self._create_iotcc_json()
+        self._iotcc_json_load_retry = int(read_liota_config('IOTCC', 'iotcc_load_retry'))
```

Comment:
In my testing, I figured out I`m getting the following error after loading the package "iotcc_mqtt.py"
 Exception in initialization: No section: 'IOTCC'. Can you please look into the issue as I need to merrge my PR on top of this.